lib/functional.fnl

```diff
@@ -237,6 +243,7 @@
  :eq?       eq?
  :filter    filter
  :find      find
+ :first     first
```

Comment:
We can turn this table into the shorthand `: first` version since you're touching it anyway.